ckcp/hack/util/util-development-patch.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The Pipeline Service Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+update_git_reference() {
+    local USER_GIT_HANDLE=$(echo "$1" | grep -Eo '([A-Za-z0-9-]+)' | sed '4q;d')
+    local REF="$2"
+    local file="$3"
+
+    sed -i -e "s/openshift-pipelines/$USER_GIT_HANDLE/g" "$file"
```

Comment:
The user could rename their repository, so changing just the handle is not enough.

Reply:
yeah agreed, thanks for the suggestion.